browser.js

```diff
@@ -10,22 +10,47 @@ const urls = {
 	v6: 'https://ipv6.icanhazip.com/'
 };
 
+const backupUrls = {
```

Comment:
I think we should use the wording `fallback` instead of `backup` (everywhere).

Reply:
why not an array like

```js
const fallbackUrlsV4 = ['https://api.ipify.org']
const fallbackUrlsV6 = ['https://api6.ipify.org']
```